query/common/src/main/java/org/codice/ddf/admin/common/fields/base/scalar/BooleanField.java

```diff
@@ -32,6 +32,10 @@ public BooleanField(String fieldName) {
     this(fieldName, null, null);
   }
 
+  public BooleanField(String fieldName, String description) {
```

Comment:
✏️ I don't think this will do anything. You can only add a description to a custom field type, other wise the default boolean description in graphql-java is used if I remember correctly.

Reply:
Just tried to add a description and this is true :( I'll remove these.